WebDevStudios/nextjs-wordpress-starter
# Feature/369 code reorganization gf

References #369

### Description

Updates GF handling to work with WP GraphQL Gravity Forms v0.4.
Refactors GF form submission to use WP GraphQL instead of REST API.
**Removes** support for FileUploadField as media uploads are currently not supported by WP GraphQL and WP GraphQL Gravity Forms. :disappointed: (New issue to re-add once uploads are supported: #380)

**Note: This PR requires the [BE PR](https://github.com/WebDevStudios/wds-headless-wordpress/pull/22) to be merged first.**

### Verification

https://nextjs-wordpress-starter-7ibw9etl3-webdevstudios.vercel.app/contact
1. Fill out and submit form.
2. Form should display success message.
3. Go to admin and confirm entry displays as expected.
